packages/docs/blog/2021-07-09-remotion-2-2.md

```diff
@@ -0,0 +1,165 @@
+
+We added support for environment variables from the CLI, using a `.env` file, and allowing you to pass variables via our Node.JS APIs too. [Click here to learn more.](/docs/env-variables)
+
+## `<Freeze />` component
+
+This new core component will freeze all of it's children and make them think that the video is paused at a certain time.
+
+Use it for example to display a still frame from a video:
+
+```tsx twoslash title="MyVideo.tsx"
+import {Freeze, Video} from 'remotion';
+import myVid from './vid.mp4';
+
+export const MyVideo = () => {
+  return (
+    <Freeze frame={30}>
+      <Video src={myVid} />
+    </Freeze>
+  )
+}
+```
+
+However, it works for any content. As you can see the API is dead simple. You can combine it with the [`<Sequence>`](/docs/sequence) API to make any content play, pause, and then continue again.
+
+[Read more about the `<Freeze />` component here.](/docs/freeze)
+
+## Video + Audio `playbackRate`
+
+This prop allows you to slow down or speed up video and audio elements! Now you can import a video in normal speed and play it in slow motion or timelapse without rerendering the video.
+
+```tsx twoslash title="LightningSpeed.tsx"
+import {Video} from 'remotion';
+import myVid from './vid.mp4';
+
+export const MyVideo = () => {
+  return (
+    <Video src={myVid} playbackRate={4}/>
+  )
+}
+```
+
+While previewing, we are using the HTML5 `playbackRate` API, and when rendering, we will calculate the correct FFMPEG command for any tempo.
+
+Learn more about the `playbackRate` props for [`<Audio />`](/docs/audio/#controlling-playback-speed) and [`<Video />`](/docs/video/#controlling-playback-speed).
+
+## `@remotion/three` package
+
+In case you missed it, we released a new helper package for [React Three Fiber](https://github.com/pmndrs/react-three-fiber)!
+
+<ThreeDPlayer />
+<br/>
+
+Three.JS is admittedly not easy, but with the work that [Poimandres](https://github.com/pmndrs) is doing in making it more approachable and more integrated with React, this is changing rapidly. Creating 3D videos in React and rendering them to real MP4 videos, we are making it possible!
+
+Learn more about [`@remotion/three`](/docs/three) and get started using our [template](https://github.com/remotion-dev/template-three).
+
+## ProRes support and MKV support
+
+Creating MP4 videos with React, that's just our marketing tagline. We support a lot of other codecs too, [like H265, WebM even transparent ones!](/docs/encoding)
+
+We have now added two more:
+
+- ProRes is a codec suitable for video editors using Final Cut Pro, Adobe Premiere or Davinci Resolve. Say you want to create an overlay animation using Remotion and use it in a traditional video cutting program, you can now use ProRes to export the video with an alpha channel and import it many other programs losslessly.
```

Comment:
[...], you can now use ProRes to export the video with an alpha channel and import it *to* many other programs losslessly.